index.html

```diff
@@ -45,11 +46,28 @@
           alt="triangle with all three sides equal"
           height="87"
           width="100"
+          onclick="handleClick()"
+        />
+
+        <img
+          class="menu-close"
+          src="./images/close.svg"
+          alt="triangle with all three sides equal"
+          height="87"
+          width="100"
+          onclick="handleClick()"
         />
+
+        <div class="menu-btn">
+          <div class="line-1"></div>
+          <div class="line-2"></div>
+          <div class="line-3"></div>
+        </div>
+
         <div class="menu-buttons">
-          <a href="/" class="menu-button">Portfolio</a>
-          <a href="/" class="menu-button">About</a>
-          <a href="/" class="menu-button">Contact</a>
+          <a href="#landing" class="menu-button">Portfolio</a>
+          <a href="#about" class="menu-button">About</a>
+          <a href="#contact" class="menu-button">Contact</a>
```

Comment:
* when i click on the mobile menu option the mobile menu should disappear as stated in the project requirement, please make the necessary adjustment

Reply:
Thanks, Davidosky007. I missed out on that detail.
Then I realized that the menu buttons were lacking some features. So I added some js code to make them close the modal when the button is clicked and the page automatically scrolls to the section desired.
For the desktop version, I added some more js code, so now, when the user scrolls over the different sections, the button corresponding to that section gets underlined.